azure/table_azure_compute_virtual_machine.go

```diff
@@ -237,6 +237,12 @@ func tableAzureComputeVirtualMachine(_ context.Context) *plugin.Table {
 				Type:        proto.ColumnType_BOOL,
 				Transform:   transform.FromField("VirtualMachineProperties.AdditionalCapabilities.UltraSSDEnabled"),
 			},
+			{
+				Name:        "vhd_uri",
```

Comment:
@sayan133 @LalitTurbot What is our strategy here on determining the column name? It seems like `VirtualMachineProperties` has different properties with different depths. In this case, for `vhd_uri`, Terraform has a field for it, but others like `require_guest_provision_signal` don't. Another example is for `linux_configuration_ssh_public_keys`, we include `linux_configuration_ssh`, but for `provision_vm_agent` we don't.